xendit-java-lib/src/main/java/com/xendit/model/DisbursementClient.java

```diff
@@ -19,16 +19,21 @@ public Xendit.Option getOpt() {
     return opt;
   }
 
-  public Disbursement create(Map<String, Object> params) throws XenditException {
-    return createRequest(new HashMap<>(), params);
+  public DisbursementIDR createIDRDisbursement(Map<String, Object> params) throws XenditException {
```

Comment:
this will become a breaking change. @xenditjoshjimenez @hglohika 
Can we have one function and then based on currency we can forward the request

Reply:
ah dang, yeah since we changed the initial signature this is a breaking change as well.

@itzpranz @hglohika any idea on how we can do the internal forwarding? java has static typing, and we need to map the response of the API to its own class (idr -> disbursement idr class model, php -> disbursement php class model)

one approach that comes into mind is to just have both idr and php fields on a single class model, where some are optional. but that might make the sdk harder to use + read

Reply:
Better create a new Java class called DisbursmentPHClient. 
So later when we will release disbursement v2 for customer we can name it DisbursementV2Client